test/integration/kubernetes_deploy_test.rb

```diff
@@ -252,23 +252,42 @@ def test_invalid_k8s_spec_that_is_valid_yaml_but_has_no_template_path_in_error_p
     ], in_order: true)
   end
 
-  def test_bad_container_image_on_run_once_halts_and_fails_deploy
+  def test_bad_container_image_on_unmanaged_pod_halts_and_fails_deploy
```

Comment:
This test didn't test what it claimed to. I fixed it and added the test below to replace the thing this one was actually testing.